src/main/java/de/doubleslash/keeptime/viewpopup/SearchPopup.java

```diff
@@ -0,0 +1,262 @@
+package de.doubleslash.keeptime.viewpopup;
```

Comment:
info: will validate this class functionality wise in other PR where it was updated